libs/features/pages/about/src/lib/components/sdg-section/sdg-section.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+import { InfoCard } from '@elewa-website/models/schema/ui/cards';
+import { __highlightedSDGList } from '@elewa-website/data/sections';
+
+@Component({
+  selector: 'elewa-website-sdg-section',
+  templateUrl: './sdg-section.component.html',
+  styleUrls: ['./sdg-section.component.scss'],
+})
+export class SdgSectionComponent {
+  @Input() sdgList: InfoCard[] = __highlightedSDGList;
+  
+  buttonData = {
+    text: 'Unlock your potential',
+    bgColor: 'transparent'
+  }
```

Comment:
For purposes of keeping the component clean and ensuring readability and maintainability, the 'buttonData' variable content should be moved to the Typescript file containing the '__highlightedSDGList' and referenced from there.

Reply:
@AnnaL001 I've made the requested changes